src/routes/webhooks/customerio.ts

```diff
@@ -208,8 +210,10 @@ export const customerio = async (fastify: FastifyInstance): Promise<void> => {
       const payload = req.body;
 
       if (subscriptionMetrics.includes(payload.metric)) {
-        const con = await createOrGetConnection();
-        await syncSubscription(payload.data.identifiers.id, con);
+        pushToRedisList(
```

Comment:
Additionally, redis can always get cleared at any point so if that happens we lose some subscriptions to sync? Giant edge case but just so we are aware of it?

Reply:
I also raised this one before we started. It needs to probably be validated by @idoshamun we ok with it. 
(As it "never" happened so far "knock on wood")

Reply:
I'd say it's a quick for to stop the alerts and 500 errors we are currently getting lots of.
@rebelchris you also mentioned that @sshanzel might implement temporal? Once that's done, we could probably hook into that.

Till then I think using redis is fine, it hadn't been cleared for ages (years even) until we downscaled it. And we're talking about loss of possibly 5 minutes

Reply:
I agree with the compromise @omBratteng suggests. We should reconsider once we have a queue system until then let's go with redis. I appreciate everyone's concern to keep the data in tact though 🙏